api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -294,9 +294,14 @@ public void setUp() {
     WorkbenchConfig testConfig = new WorkbenchConfig();
     testConfig.cohortbuilder = new WorkbenchConfig.CohortBuilderConfig();
     testConfig.cohortbuilder.enableListSearch = false;
+    testConfig.firecloud = new WorkbenchConfig.FireCloudConfig();
```

Comment:
You can add this at L250 instead of exposing the configProvider like this.

The cohortReviewController setter also shouldn't be necessary, but that's an existing issue which I wouldn't advise digging into in this PR.

Reply:
i think both are necessary bc of how we are calling the config in the controller